packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -815,11 +830,36 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 		}
 
 		toggleMoreComponents () {
+			if (this.state.hasReadInfoComponents) {
+				this.focusMoreButton();
+			}
+
 			this.setState((prevState) => {
 				return {
 					showMoreComponents: !prevState.showMoreComponents
 				};
 			});
+
+		}
+
+		focusMoreButton = () => {
+			if (!this.props.visible) {
+				return;
+			}
+
+			const selectedButton = Spotlight.getCurrent();
+			if (selectedButton === this.moreButtonNode) {
+				// Press enter of 5way key
+				selectedButton.blur();
+				Spotlight.focus(selectedButton);
```

Comment:
any reason why we use `Spotlight.focus` instead of `selectedButton.focus()`? Isn't the purpose of this block only to reset the focus for a11y?